gradle.properties

```diff
@@ -13,11 +13,12 @@ commonsCLIVersion=1.4
 commonsCSVVersion=1.8
 commonsCodecVersion=1.14
 commonsMath3Version=3.6.1
-grizzlyVersion=3.1.3
+grizzlyVersion=2.35
```

Comment:
why we are going to lower version here  @dada-dell-emc  ?

Reply:
Because of higher version there is conflicting versions dependency on glassfish library.